src/config.ts

```diff
@@ -1,10 +1,11 @@
 import 'dotenv/config';
-import {resolve} from 'node:path';
+import {dirname, resolve} from 'node:path';
+import {fileURLToPath} from 'node:url';
 
 export type Config = {
     cacheDir: string;
 };
 
 export const config = {
-    cacheDir: process.env.CACHE_DIR ?? resolve(import.meta.dirname, '..', '_cache'),
+    cacheDir: process.env.CACHE_DIR ?? resolve(dirname(fileURLToPath(import.meta.url)), '..', '_cache'),
```

Comment:
It's probably OK to drop the `dirname` call here, but I'm using what I know works, LOL...